scripts/dropstat

```diff
@@ -117,10 +117,10 @@ class DropStat(object):
         """
 
         try:
-            pickle.dump(self.get_counts_table(self.gather_counters(std_port_rx_counters + std_port_tx_counters, DEBUG_COUNTER_PORT_STAT_MAP), COUNTERS_PORT_NAME_MAP),
-                        open(self.port_drop_stats_file, 'wb+'))
-            pickle.dump(self.get_counts(self.gather_counters([], DEBUG_COUNTER_SWITCH_STAT_MAP), self.get_switch_id()),
-                        open(self.switch_drop_stats_file, 'wb+'))
+            json.dump(self.get_counts_table(self.gather_counters(std_port_rx_counters + std_port_tx_counters, DEBUG_COUNTER_PORT_STAT_MAP), COUNTERS_PORT_NAME_MAP),
+                        open(self.port_drop_stats_file, 'w+'))
+            json.dump(self.get_counts(self.gather_counters([], DEBUG_COUNTER_SWITCH_STAT_MAP), self.get_switch_id()),
```

Comment:
>json.dump [](http://example.com/codeflow?start=12&length=9)

Do you need `default=json_serial` in these 2?

Reply:
I don't think so, `json_serial` is serializer for time objects but dropstat does not have `time` key.